src/main/kotlin/no/nav/familie/ba/sak/kjerne/behandlingsresultat/BehandlingsresultatUtils.kt

```diff
@@ -24,6 +31,57 @@ object BehandlingsresultatUtils {
             message = "Kombiansjonen av behandlingsresultatene $behandlingsresultater er ikke støttet i løsningen."
         )
 
+    internal fun erEndringIBeløp(
+        nåværendeAndeler: List<AndelTilkjentYtelseMedEndreteUtbetalinger>,
+        forrigeAndeler: List<AndelTilkjentYtelseMedEndreteUtbetalinger>,
+        personerFremstiltKravFor: List<Aktør>
+    ): Boolean {
+        val allePersonerMedAndeler = (nåværendeAndeler.map { it.aktør } + forrigeAndeler.map { it.aktør }).distinct()
+
+        val erEndringPerPerson = allePersonerMedAndeler.map { aktør ->
+            erEndringIBeløpForPerson(
+                nåværendeAndeler = nåværendeAndeler.filter { it.aktør == aktør },
+                forrigeAndeler = forrigeAndeler.filter { it.aktør == aktør },
+                opphørstidspunkt = nåværendeAndeler.maxOf { it.stønadTom },
+                erFremstiltKravForPerson = personerFremstiltKravFor.contains(aktør)
+            )
+        }
+
+        return erEndringPerPerson.any { it }
```

Comment:
Lurer på om vi bare kan hente ut boolean'n med engang slik at vi slipper å mappe også any. Noe ala

```suggestion
        val erEndringPåMinstEnPerson = allePersonerMedAndeler.any { aktør ->
            erEndringIBeløpForPerson(
                nåværendeAndeler = nåværendeAndeler.filter { it.aktør == aktør },
                forrigeAndeler = forrigeAndeler.filter { it.aktør == aktør },
                opphørstidspunkt = nåværendeAndeler.maxOf { it.stønadTom },
                erFremstiltKravForPerson = personerFremstiltKravFor.contains(aktør)
            )
        }

        return erEndringPåMinstEnPerson
```

Reply:
Good catch! Endrer 👍